junyub/code/mcdonalds.py

```
@@ -0,0 +1,166 @@
+from selenium import webdriver
+from selenium.webdriver.chrome.service import Service
+from selenium.webdriver.chrome.options import Options
+from selenium.webdriver.common.by import By
```

Ruff (0.12.2)

`selenium.webdriver.common.by.By` imported but unused

Remove unused import: `selenium.webdriver.common.by.By`

(F401)